src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1456,6 +1456,11 @@ public void Dispose()
             {
                 SearchModel.ItemProduced -= SearchModel_ItemProduced;
             }
+            //handle the disposal of IDSDK manager instance for sandbox
+            if(AuthenticationManager.AuthProvider is IDSDKManager)
```

Comment:
you can do 
```  if(AuthenticationManager.AuthProvider is IDSDKManager idsdkProvider){
idsdkProvider.dispose
}
```
using pattern matching

Reply:
could either of these properties be null?
AuthenticationManager, AuthProvider ?

Reply:
No, still add some checks, like?
```
if(AuthenticationManager?.AuthProvider is IDSDKManager idsdkProvider){
    idsdkProvider?.Dispose()
}
```